pkg/security/probe/probe_windows.go

```diff
@@ -49,7 +49,9 @@ type WindowsProbe struct {
 
 // Init initializes the probe
 func (p *WindowsProbe) Init() error {
-	pm, err := procmon.NewWinProcMon(p.onStart, p.onStop, p.onError)
+	// the last two parameters are buffer size and number of buffers to supply the driver
+	// for process notifications.  We can override with configurables if desired.
+	pm, err := procmon.NewWinProcMon(p.onStart, p.onStop, p.onError, 0, 0)
```

Comment:
suggestion: could we export `procmonReceiveSize` and `procmonNumBufs` (renamed to `DefaultReceiveSize` and `DefaultNumBufs`) and pass them here instead of `0,0`? That'd make it more obvious that we're using the default and removes the need to have an additional check in `NewWinProcMon` for the `0` values.

```suggestion
	pm, err := procmon.NewWinProcMon(p.onStart, p.onStop, p.onError, procmon.DefaultReceiveSize, procmon.DefaultNumBufs)
```